config/local/apiml-service.yml

```diff
@@ -20,14 +20,38 @@ server:
         # redisratelimiter: TRACE
 
 apiml:
+    health:
+        protected: false
     gateway:
         servicesToLimitRequestRate: discoverableclient
         cookieNameForRateLimiter: apimlAuthenticationToken
     security:
+        x509:
+            enabled: true
+            acceptForwardedCert: true
```

Comment:
This value should be used only if additional registration is done (see multitenancy).